src/views/tables/PositionsTable/PositionsActionsCell.tsx

```diff
@@ -127,14 +127,16 @@ export const PositionsActionsCell = ({
           />
         </WithTooltip>
       )}
-      <$TriggersButton
-        key="share"
-        onClick={openShareDialog}
-        iconName={IconName.Share}
-        shape={ButtonShape.Square}
-        disabled={isDisabled}
-        $uiRefreshEnabled={uiRefresh}
-      />
+      <WithTooltip tooltipString={stringGetter({ key: STRING_KEYS.SHARE })}>
+        <$TriggersButton
+          key="share"
+          onClick={openShareDialog}
+          iconName={IconName.Share}
+          shape={ButtonShape.Square}
+          disabled={isDisabled}
+          $uiRefreshEnabled={uiRefresh}
+        />
+      </WithTooltip>
       {showClosePositionAction && (
         <WithTooltip tooltipString={stringGetter({ key: STRING_KEYS.CLOSE_POSITION })}>
           <$CloseButtonToggle
```

Comment:
might be my bad + unrelated to this pr but i realized the toggled state looks a little off because there's no padding t_t, can we add back padding for this?
<img width="142" alt="image" src="https://github.com/user-attachments/assets/c42e50b3-1632-4a46-a4d7-c57a28a6bba4">

Reply:
we probably don't need a toggle state for this imo

Reply:
oh good point heh lemme fix